### PR TITLE
Added <strike> support

### PR DESCRIPTION
**Pull request by SuperPichu**
Your list says the default TextView supports this, but it does not, so I added support

### CODE REVIEW

File: src/main/java/org/sufficientlysecure/htmltextview/HtmlTagHandler.java [hunk 1]
```diff
@@ -43,6 +43,9 @@
 
     private static class Center {
     }
+	
+	private static class Strike{
+	}
```
Comment (dschuermann) [inline src/main/java/org/sufficientlysecure/htmltextview/HtmlTagHandler.java]: Please use spaces instead of tabs here :)

**Comment by dschuermann** [comment]
Please spaces instead of tabs and add whitespace before opening clauses.

**Comment by SuperPichu** [comment]
What do you mean by add whitespace?

**Comment by dschuermann** [comment]
I was just nitpicking on coding style, not that important :) . I meant
``else if (tag.equalsIgnoreCase("s") || tag.equalsIgnoreCase("strike")) {`` instead of
``else if (tag.equalsIgnoreCase("s") || tag.equalsIgnoreCase("strike")){``

I merged it.

**Pull request merged**